types/setting.go

```diff
@@ -50,6 +50,7 @@ const (
 	SettingNameReplicaAutoBalance                                       = SettingName("replica-auto-balance")
 	SettingNameStorageOverProvisioningPercentage                        = SettingName("storage-over-provisioning-percentage")
 	SettingNameStorageMinimalAvailablePercentage                        = SettingName("storage-minimal-available-percentage")
+	SettingNameStorageReservedPercentage                                = SettingName("storage-reserved-percentage-for-default-disk")
```

Comment:
Please change all variables accordingly instead of updating the option.

Reply:
Updated. Thanks.